AirSim/PythonClient/airsim/client.py

```diff
@@ -1055,7 +1038,7 @@ def getMultirotorState(self, vehicle_name = ''):
 
 # -----------------------------------  Car APIs ---------------------------------------------
 class CarClient(VehicleClient, object):
-    def __init__(self, ip = "", port = 41451, timeout_value = 3600):
+    def __init__(self, ip = "", port = 41451, timeout_value = 10):
```

Comment:
getting a timeout after 5 minutes is insane. Nobody got time for that. Setting it to a realistic 10 seconds.